public/index.html

```diff
@@ -2,7 +2,6 @@
 <html lang="en">
```

Comment:
![image](https://github.com/lorenzoworx/math/assets/93039359/54cfb244-665d-43c4-bd1a-83986a99feec)

- [ ] It would be nice if you remove all the text, images, and additional styles from your project as this public folder should only have an **index.html** and everything else should be removed to follow the project requirements :blush: